src/main/java/marquez/core/mappers/GetNamespaceResponseMapper.java

```diff
@@ -1,21 +1,19 @@
 package marquez.core.mappers;
 
-import java.util.Optional;
+import static java.util.Objects.requireNonNull;
+
 import marquez.api.GetNamespaceResponse;
 import marquez.core.models.Namespace;
 
-public class GetNamespaceResponseMapper implements Mapper<Namespace, GetNamespaceResponse> {
-
-  CoreNamespaceToApiNamespaceMapper namespaceMapper = new CoreNamespaceToApiNamespaceMapper();
-
-  @Override
-  public Optional<GetNamespaceResponse> map(Namespace namespace) {
-    if (namespace == null) {
-      return Optional.empty();
-    }
+// TODO: Move to marquez.api.mappers pgk
```

Comment:
are these `TODO`s still valid?

Reply:
Yeah, I was thinking to add all model mappers that are API specific to the `marquez.api.mappers` pkg and keep only core mappers in `marquez.core.mappers`. Thoughts?

Reply:
I like that idea.